packages/test/test-service-load/src/utils.ts

```diff
@@ -62,6 +62,9 @@ class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
             event.category = event.testCategoryOverride;
         }
 
+        if (typeof event.message === "string" && event.message.indexOf("FaultInjectionNack") > -1) {
```

Comment:
This is hacky but the injection and this logger logging the error is in same test package, so I though we can do it without touching any other packages.

Reply:
seems ok. i think i explored something similar but didn't do it because i was hacky :) but i think it's the right thing for now. maybe the only improvement would be to make this an else case with the above